### PR TITLE
Create contributing.md

### PR DESCRIPTION
**Pull request by rjchow**
Create contributing.md to serve as guidelines for open source contributions for the community, in order to set standards and expectations right.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+-   For english language contributions: Has a good score on [Grammarly](https://gist.github.com/PurpleBooth/grammarly.com) or [Hemingway App](http://www.hemingwayapp.com/)
+-   For code contributions: It should minimally pass the automated code quality checks such as linting and tests, as well was having new tests of its own.
+
+### Does it move this repository closer to my vision for the repository
+
+The aim of this repository can be found in [README.md](README.md)
+
+### Does it follow the contributor covenant
+
+This repository has a [code of conduct](CODE_OF_CONDUCT.md), This repository has a code of conduct, I will remove things that do not respect it.
```
Comment (Nebulis) [inline CONTRIBUTING.md:59]: We don't

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+### Make your update:
+Make your changes to the file(s) you'd like to update. 
+  - Are you making changes to the application code? See [README.md](README.md) for more information on how to run this project locally.
+  - Are you contributing to documentation in this repository? We use [GitHub Markdown](contributing/content-markup-reference.md).
+
+### Open a pull request
+When you're done making changes and you'd like to propose them for review, use the pull request button to open your PR (pull request).
+
+### Submit your PR & get it reviewed
+- Once you submit your PR, core contributors from this repository will review it with you. The first thing you're going to want to do is a [self review](#self-review).
```
Comment (Nebulis) [inline CONTRIBUTING.md:88]: Should we link to team page or add names to contact in case ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (rjchow) [reply CONTRIBUTING.md:59]: oh yea forgot to add it, will add it

File: CONTRIBUTING.md (hunk 2 above)
Comment (rjchow) [reply CONTRIBUTING.md:88]: in case of?

Comment (Nebulis) [reply CONTRIBUTING.md:88]: problems to open PR / question / dunno

**Comment by rjchow** [comment]
nah 😆
if github is down i don’t want to be dealing with it
if they don’t know how to do a pr they can read this document again and
follow the links

File: CONTRIBUTING.md (hunk 2 above)
Comment (rjchow) [reply CONTRIBUTING.md:88]: nah if github is down i don’t want to be dealing with it
if they don’t know how to do a pr they can read this document again and
follow the links

**Pull request merged**